### DISCUSSION TITLE
About "content"  in our Tailwind CSS Configuration

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#12655] [Help]
Greetings, i am trying to start my server by using npm run dev in vs code but everytime i run that command a warning shows that " No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration" **and i checked the content again and again there is no mistake because i have copied the content directly through Tailwind CSS website** please try to help me out in this matter thank you in advance

**Comment by wongjn** [comment]
Consider checking:

- You have used at least one utility class in a file covered by the `content` file globs.
- The `content` file globs you copied from the Tailwind website are relevant to your project.
